controllers/storagecluster/reconcile.go

```diff
@@ -107,7 +107,7 @@ var validTopologyLabelKeys = []string{
 }
 
 // +kubebuilder:rbac:groups=ocs.openshift.io,resources=*,verbs=get;list;watch;create;update;patch;delete
-// +kubebuilder:rbac:groups=ceph.rook.io,resources=cephclusters;cephblockpools;cephfilesystems;cephnfses;cephobjectstores;cephobjectstoreusers;cephrbdmirrors,verbs=*
+// +kubebuilder:rbac:groups=ceph.rook.io,resources=cephclusters;cephblockpools;cephfilesystems;cephnfses;cephobjectstores;cephobjectstoreusers;cephrbdmirrors;cephBlockPoolRadosNamespace,verbs=*
```

Comment:
Code changes should try to keep new items formatted in the same way as earlier items. This is stil in cAmElCaP, but everything else is allinlowercase. Also, this is not plural, where all other items are. Please fix both, like so.

```suggestion
// +kubebuilder:rbac:groups=ceph.rook.io,resources=cephclusters;cephblockpools;cephfilesystems;cephnfses;cephobjectstores;cephobjectstoreusers;cephrbdmirrors;cephblockpoolradosnamespaces,verbs=*
```